src/hypermedia-constants.js

```diff
@@ -162,6 +162,9 @@ export const Rels = {
 	},
 	Users: {
 		canonicalUser: 'https://users.api.brightspace.com/rels/canonical-user'
+	},
+	IANA: {
+		edit: 'edit'
```

Comment:
I believe we want to be moving towards having `rels` and `classes` defined directly in the entity that uses them rather than in `hypermedia-constants`. That being said, since IANA Rels are a set of standardized relationships I think it makes sense to keep them here. @m6jones or @ctwomey1 _may_ have some additional context.